Makefile.am

```diff
@@ -32,6 +31,7 @@ EXTRA_DIST =					\
 	gpg_uid					\
 	nginx_version				\
 	version-gen.sh
+dist_pkgdata_DATA = COPYING
```

Comment:
Is `dist_` needed?

Reply:
It seems that no need to `dist_` prefix.

Reply:
Thanks.